src/test/java/bio/terra/janitor/app/controller/JanitorApiControllerTest.java

```diff
@@ -41,29 +46,26 @@ public void setup() {
   }
 
   @Test
-  public void testCreateResourceSuccess() throws Exception {
-    CloudResourceUid cloudResourceUid = newGoogleProjectResourceUid();
-    this.mvc
-        .perform(
-            post("/api/janitor/v1/resource")
-                .contentType(MediaType.APPLICATION_JSON)
-                .content(newJsonCreateRequestBody(cloudResourceUid, Optional.of(DEFAULT_LABELS))))
-        .andDo(MockMvcResultHandlers.print())
-        .andExpect(status().isOk());
+  public void createResourceSuccess() throws Exception {
+    String response =
+        this.mvc
+            .perform(
+                post("/api/janitor/v1/resource")
+                    .contentType(MediaType.APPLICATION_JSON)
+                    .content(
+                        newJsonCreateRequestBody(
+                            newGoogleProjectResourceUid(), Optional.of(DEFAULT_LABELS))))
+            .andDo(MockMvcResultHandlers.print())
+            .andExpect(status().isOk())
+            .andReturn()
+            .getResponse()
+            .getContentAsString();
 
-    // Verify data are created correctly.
-    assertCreateResultMatch(
-        cloudResourceUid,
-        GOOGLE_PROJECT,
-        Optional.empty(),
-        Optional.empty(),
-        TIME_TO_LIVE_MINUTE,
-        jdbcTemplate,
-        DEFAULT_LABELS);
+    assertResourceExists(response);
   }
 
   @Test
-  public void testCreateResourceFail_missingField() throws Exception {
+  public void createResourceFail_missingField() throws Exception {
```

Comment:
as-is, it's not very obvious what the missing field is. inlining the request creation or adding a comment would help.

Reply:
Did both